src/app/context/context.component.ts

```diff
@@ -4,9 +4,9 @@ import { MatDialog } from '@angular/material/dialog';
 import { DialogInfoComponent } from '../dialog-info/dialog-info.component';
 import { MatTableDataSource } from '@angular/material/table';
 
+import { ActivityService } from '../activity.service';
 import { Utils } from '../utils';
-import { interval, Observable, Subscription } from 'rxjs';
-import allContextDecksCollection from '../../assets/activities/contextarray.json';
+import { interval, Subscription } from 'rxjs';
 
 interface ContextModel {
```

Comment:
contect.model.ts-be menjenek ki az interfaceek innen.

Reply:
done for all activities